Dockerfile

```diff
@@ -1,18 +1,34 @@
 # stage 1 Build qgb binary
-FROM golang:1.20-alpine as builder
+FROM --platform=$BUILDPLATFORM docker.io/golang:1.21-alpine3.18 as builder
 RUN apk update && apk --no-cache add make gcc musl-dev git
 COPY . /orchestrator-relayer
 WORKDIR /orchestrator-relayer
 RUN make build
 
 # final image
-FROM alpine:3.18.3
+FROM --platform=$BUILDPLATFORM docker.io/alpine:3.18.3
+
+ARG UID=10001
+ARG USER_NAME=celestia
```

Comment:
Basically, we are aligning all DockerFiles into 1 style like we have in app and node, where there is a celestia user and not root anymore. Let me know if this is fine for the orchestrator-relayer